.github/workflows/test-linux.yml

```diff
@@ -36,11 +36,20 @@ on:
       - '!.github/workflows/build-subrepos.yml'
 
   workflow_dispatch:
+    inputs:
+      ssh:
+        description: 'Enable ssh debugging'
+        required: false
+        default: false
```

Comment:
What of these two entries need to be made `true` to enable SSH debugging? Please add a comment here (or elsewhere if this is not the right place) to explain that for future reference. And also do it in all workflow files.

Reply:
On the web interface, it will appear as a checkbox option just like our installer workflow:
<img width="351" alt="Screenshot 2024-05-24 at 7 36 22 AM" src="https://github.com/spyder-ide/spyder/assets/9618975/10dbe4e5-c3c5-443e-bf1b-5bb740f6da6c">

For the cli:
```
$ gh workflow run test-linux.yml --ref <branch> -f ssh=true
```

I can add the cli example as a comment to all workflows.

Reply:
On the web interface, it will appear as a checkbox option just like our installer workflow:
<img width="351" alt="Screenshot 2024-05-24 at 7 36 22 AM" src="https://github.com/spyder-ide/spyder/assets/9618975/10dbe4e5-c3c5-443e-bf1b-5bb740f6da6c">

For the cli:
```
$ gh workflow run test-linux.yml --ref <branch> -f ssh=true
```

I can add the cli example as a comment to all workflows.

Reply:
> I can add the cli example as a comment to all workflows.

Yeah, please do.